ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/CPAN/tasks/main.yml

```diff
@@ -47,6 +47,7 @@
 
 - name: Install XML::Parser
   command: cpanm --with-recommends --sudo XML::Parser
+  when: ! (ansible_distribution == "SLES" and ansible_distribution_major_version == "12")
```

Comment:
This doesnt skip, as it should, on SLES 12. It may be a syntax error.
Even if skipping this task on SLES 12 is a temp solution, I think the condition `ansible_distribution != "SLES" and ansible_distribution_major_version != "12"` would be more suitable

Reply:
Fix in 👍